docs/src/suitable_commands.md

```diff
@@ -0,0 +1,55 @@
+# Suitable commands
+
+### help
+
+### index
+
+- [x] Manage(list, add, delete) custom plugin indexes
+
+### info
+
+- [x] Show information about a plugin via NAME & INDEX/NAME
```

Comment:
Indexes would not be user facing - they will be part of the SDK IMHO
WDYT?>

Reply:
users of CLI don't need to know indexes.